modules/monitoring-about-configuring-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-configuring-core-platform-monitoring_{context}"]
+= About configuring core platform monitoring
+
+Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace to configure the following core platform monitoring components: 
+
+[source,yaml]
+----
+# Holds configuration related with the main Alertmanager instance.
+alertmanagerMain: <struct>
+# Boolean flag to enable monitoring for user-defined projects.
+# Default: false
+enableUserWorkload: <bool>
+# Holds configuration settings for the Grafana component
+grafanaConfig: <struct>
+# Holds configuration related to the prometheus-adapter
+k8sPrometheusAdapter: <struct>
+# Holds configuration related with kube-state-metrics agent
+kubeStateMetrics: <struct>
+# Holds configuration related with the prometheus component that will be managed by
+# the prometheus-operator
```

Comment:
* and to follow the standard object naming and special case conventions... e.g.,
```suggestion
# the Prometheus Operator.
```